internal/pkg/cli/config/config.go

```diff
@@ -75,7 +75,7 @@ type Config struct {
 func Default() Config {
 	cfg := Config{}
 	cfg.Config = slicerConfig.Default()
-	cfg.MemoryLimit = 256 * datasize.MB
+	cfg.MemoryLimit = 512 * datasize.MB
```

Comment:
Read ahead approach needs more memmory.

A low limit will cause frequent GC calls and reduced speed.
​